store/src/java/com/zimbra/cs/service/admin/FlushCache.java

```diff
@@ -328,4 +328,9 @@ private static String sanitizeImapCacheTypes(String cacheTypes) {
         }
         return imapTypes.isEmpty() ? null : Joiner.on(",").join(imapTypes);
     }
+
+    public static void flushConfigCacheOnAllServers(ZimbraSoapContext zsc, FlushCacheRequest req) throws ServiceException {
```

Comment:
Since it's named to flush the cache of the config only then how will it prevent another type of cache flush?